setup.py

```diff
@@ -27,7 +27,7 @@
         ],
     },
     tests_require=[
-        'mock',
+        'mock==1.0.1',
```

Comment:
please don't require a specific version. greater-equals is fine if a feature was introduced in a particular version of mock.